lib/seed/blood_pressure_seeder.rb

```diff
@@ -108,8 +127,10 @@ def call
 
       encounter_result = Encounter.import(encounters)
       observation_result = Observation.import(observations)
+      prescription_drug_result = PrescriptionDrug.import(prescription_drugs)
       counts[:encounter] = encounter_result.ids.size
       counts[:observation] = observation_result.ids.size
+      counts[:prescription_drug] = prescription_drug_result.ids.size
```

Comment:
You can assert on this in a spec to make sure these are created.  We have a `test` mode for seeding so that everything is deterministic, lemme know if you need help w/ that piece.